### DISCUSSION TITLE
Video cannot be played online, but can be played with Potplayer.

### DISCUSSION THREAD

**Discussion by starlight02** [spencerwooo/onedrive-vercel-index#370] [Q&A] [answered]
部署后所有的视频都无法播放，视频是 flv 格式，是不支持此格式吗？

地址：[传送门](https://cloud.aili.moe/%F0%9F%8E%99%E5%98%89%E7%84%B6%E7%9A%84%E6%AD%8C%E6%9B%B2/%5B2021-05-28%5D%5Bav760787326%5D%E3%80%8AHopeful_Dreamer%E3%80%8B%E4%B8%AA%E4%BA%BAMV%20%E6%AD%A3%E5%BC%8F%E7%89%88.flv)

**Comment by myl7** [comment, marked as answer]
Ref: #361
这个播放器支持 flv，但是需要一些额外的步骤

**Reply by starlight02** [reply]
太好了，期待合并！

**Comment by starlight02** [comment]
更新后已经可以正常播放了。
